client/js/models/help.js

```diff
@@ -0,0 +1,33 @@
+var intro = require('intro.js')
+
+var getIntro = function () {
+  const _intro = intro.introJs()
+  _intro.setOptions({
```

Comment:
i think it would be nicer to follow an underscore free naming scheme here, like for example:

```js
var introJS = require('intro.js').introJS
const intro = introJS()
intro.setOptions({ ... })
```